data/com.endlessm.HackToolbox.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessm.HackToolbox.desktop</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPL-2.1</project_license>
+  <categories>
+    <category>Game</category>
+  </categories>
```

Comment:
I don't think you need to add categories. This is not supposed to be visible to the uuser anyway.